### ISSUE TITLE
Support for Brazilians

### ISSUE THREAD

**Issue by Higor-Matos** [AdguardTeam/AdGuardDNS#257]
Hello, I ask for support to Brazilians in the acquisition of Adguard DNS and if possible "discount" for betas that contribute.

I am willing to help and be part of the Adguard team because it is a company of great affection and esteem of mine.

![image](https://user-images.githubusercontent.com/54960146/170543793-d2667a50-d60b-414d-bbe6-92bf635455a9.png)
<img width="418" alt="image" src="https://user-images.githubusercontent.com/54960146/170544072-40474613-d108-42a5-9937-4ecec8baf7b2.png">

**Comment by ameshkov** [comment]
Yeah, we'll add tariffs for more currencies including BRL. I'll keep this issue open for now.

**Comment by GulnazSalakhova** [comment]
Brazilian Real (BRL) is now available as a supported currency on the AdGuard DNS pricing page, so the main request has been fulfilled. We are closing this issue as completed.
